controller/users.go

```diff
@@ -82,7 +82,7 @@ func (c *UsersController) Show(ctx *app.ShowUsersContext) error {
 			}
 		}
 		return ctx.ConditionalRequest(*user, c.config.GetCacheControlUser, func() error {
-			return ctx.OK(ConvertToAppUser(ctx.RequestData, user, identity))
+			return ctx.OK(ConvertToAppUser(ctx.RequestData, user, identity, true))
```

Comment:
should be false, right?

Reply:
nope, should be a `true` since this is the `GET /api/user` with `Authorization..`

Reply:
My bad, this is the `/api/users/<ID>` .